### PR TITLE
docs: add Page.onceDialog for java

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: docs/src/api/class-page.md [hunk 1]
```diff
@@ -1741,6 +1741,20 @@ The page's main frame. Page is guaranteed to have a main frame which persists du
 ## property: Page.mouse
 - type: <[Mouse]>
 
+## method: Page.onceDialog
```
Comment (pavelfeldman) [inline docs/src/api/class-page.md:1744]: Let's move this to java.md?

File: docs/src/api/class-page.md [hunk 2]
```diff
@@ -1741,6 +1741,20 @@ The page's main frame. Page is guaranteed to have a main frame which persists du
...
+* langs: java
+
+Adds one-off [Dialog] handler. The handler will be removed immediately after next [Dialog] is created.
```
Comment (pavelfeldman) [inline docs/src/api/class-page.md:1747]: Let's add the code snippet here.

**Review by pavelfeldman — APPROVED** [review]

File: docs/src/api/class-page.md (hunk 1 above)
Comment (yury-s) [reply docs/src/api/class-page.md:1744]: Done.

File: docs/src/api/class-page.md (hunk 2 above)
Comment (yury-s) [reply docs/src/api/class-page.md:1747]: Done.

**Pull request merged**